Jenkinsfile

```diff
@@ -5,12 +5,29 @@ elifePipeline {
         commit = elifeGitRevision()
     }
 
-    stage 'Project tests', {
-        lock('journal--ci') {
-            builderDeployRevision 'journal--ci', commit
-            builderProjectTests 'journal--ci', '/srv/journal', ['/srv/journal/build/phpunit/*.xml', '/srv/journal/build/behat/*.xml'], ['smoke', 'project']
-        }
-    }
+    elifeOnNode(
+        {
+            stage 'Build images', {
+                checkout scm
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml build"
+            }
+
+            stage 'Project tests', {
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d"
+                dockerComposeProjectTestsParallel('journal', commit, [
+                    'phpunit': '/srv/journal/build/phpunit/*.xml',
+                    'behat': '/srv/journal/build/behat/*.xml'
+                ])
+
+                dockerComposeSmokeTests('journal', commit, [
+                    'scripts': [
+                        'cli': './smoke_tests_cli.sh',
```

Comment:
I think `cli` is exiting before the `docker-compose exec` can be run. We can define some more complex behavior like use `run` if `exec` can't be used, but to start with is it possible to put a default command like `php` or `sleep infinity`?